### ISSUE TITLE
Cannot access fr.minuskube.netherboard.api.PlayerBoard

### ISSUE THREAD

**Issue by ThatDeveloper** [MinusKube/Netherboard#20]
Hey there, in IntelliJ IDEA Ultimate when adding this api as Maven and getting an instance this happens on the set (red, alert):
Cannot access fr.minuskube.netherboard.api.PlayerBoard

```
BPlayerBoard board = Netherboard.instance().getBoard(p);
        if(board == null)
            board = Netherboard.instance().createBoard(p, "My Scoreboard");

        board.set("a", 1);
```

**Comment by ThatDeveloper** [comment]
[https://stackoverflow.com/a/40299606/12418343](https://stackoverflow.com/a/40299606/12418343)

I added also netherboard-core. Works now. Thank you.

        <dependency>
            <groupId>fr.minuskube</groupId>
            <artifactId>netherboard-core</artifactId>
            <version>1.2.0</version>
            <scope>compile</scope>
        </dependency>
        <dependency>
            <groupId>fr.minuskube</groupId>
            <artifactId>netherboard-bukkit</artifactId>
            <version>1.2.0</version>
            <scope>compile</scope>
        </dependency>

**Comment by MinusKube** [comment]
This has also been fixed in 1.2.1 :)
